packages/components/src/helperMessage/src/HelperMessage.tsx

```diff
@@ -0,0 +1,73 @@
+import { type StyledComponentProps, useStyledSystem } from "@hopper-ui/styled-system";
+import { forwardRef, type ForwardedRef, type CSSProperties, useContext } from "react";
+import clsx from "clsx";
+import { HelperMessageContext } from "./HelperMessageContext.ts";
+import { useContextProps, FieldErrorContext as RACFieldErrorContext } from "react-aria-components";
+import { InfoIcon } from "@hopper-ui/icons";
+import { type TextProps, Text } from "../../Text/index.ts";
+import { cssModule } from "../../utils/src/cssModule.ts";
+
+import styles from "./HelperMessage.module.css";
+
+export const GlobalHelperMessageCssSelector = "hop-HelperMessage";
+export interface HelperMessageProps extends StyledComponentProps<Omit<TextProps, "size">> {
+    /**
+     * Whether or not to hide the helper message icon.
+     * @default true
+     */
+    hideInfoIcon?: boolean;
```

Comment:
Thinking out loud here, should we use a more generic name such as hideDefaultIcon? This would prevent having to rename this if the icon was to change.

Reply:
hideIcon would be enough, Default is redundant